From bd87e9b33291eec35df547cfda6eac3c4b826d8b Mon Sep 17 00:00:00 2001
From: Dmitry Timoshkov <dmitry@baikal.ru>
Date: Wed, 23 Sep 2015 16:07:30 +0800
Subject: msi: Add a trace to msi_get_property.

msi_set_property already has a similar trace and this patch makes it easier
to see what is going on.

Signed-off-by: Dmitry Timoshkov <dmitry@baikal.ru>
---
 dlls/msi/package.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/dlls/msi/package.c b/dlls/msi/package.c
index 7ab2b19..01d2e89 100644
--- a/dlls/msi/package.c
+++ b/dlls/msi/package.c
@@ -2125,6 +2125,8 @@ UINT msi_get_property( MSIDATABASE *db, LPCWSTR szName,
     MSIRECORD *row;
     UINT rc = ERROR_FUNCTION_FAILED;
 
+    TRACE("%p %s %p %p\n", db, debugstr_w(szName), szValueBuf, pchValueBuf);
+
     row = msi_get_property_row( db, szName );
 
     if (*pchValueBuf > 0)
-- 
2.5.1

